.github/workflows/proto.yml

```diff
@@ -0,0 +1,26 @@
+name: Protobuf CI
+on:
+  push:
+    branches:
+      - main
+  merge_group:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+
+jobs:
+  buf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: bufbuild/buf-action@v1
+        with:
+          token: ${{ secrets.BUF_TOKEN }}
+          input: "proto"
+          breaking_against: "https://github.com/agglayer/agglayer.git#branch=main,ref=HEAD~1,subdir=proto"
```

Comment:
I don't understand the `ref` argument here. Seems `branch` should be sufficient?